### PR TITLE
Delete test issue template

### PR DESCRIPTION
**Pull request by PeterTurcan**

### CODE REVIEW

**Comment by PeterTurcan** [comment]
@alandefreitas a minor update, deleted my test template and added a short section on how to do this after the "creating an issue template" section.

File: contributor-guide/modules/ROOT/pages/version-control.adoc [hunk 1]
```diff
@@ -134,7 +136,19 @@ Here are the steps to create an issue template:
 
 Once the template is added, users who create new issues in your repository can choose to use one of your templates.
 
-Note:: Currently, the Boost Super-project does not use the GitHub Discussion feature. If filing an issue does not seem appropriate, users are encouraged to post on the https://lists.boost.org/mailman/listinfo.cgi/boost[Boost developers mailing list].
+=== How to Delete an Issue Template
```
Comment (alandefreitas) [inline contributor-guide/modules/ROOT/pages/version-control.adoc:139]: Removing the template and the sentence about GitHub discussions seems ok. But this subsection really seems weird. It has nothing to do with our audience.

If someone knows how to create an issue template, and they know that happens by including files in the repo, and they still don't know how to delete an issue template from their own repo, this person should definitely not be a boost contributor. And if they do it through the GitHub interface, as this section would suggest to be OK, that's even 100 times worse.

**Comment by alandefreitas** [comment]
>  a minor update, deleted my test template and added a short section on how to do this after the "creating an issue template" section.

Looks great. Please squash when this is complete.

**Comment by PeterTurcan** [comment]
@alandefreitas removed delete issue section, squashed, gtg

**Pull request merged**